CMSgov/mint-app
# [EASI-1899] LaunchDarkly Flag to Downgrade Assessment Team Users

# NOREF

## Changes and Description

- Created a [flag in LaunchDarkly to downgrade assessment team users](https://app.launchdarkly.com/mint/local/features/downgradeAssessmentTeam)
- Added frontend code to honor that flag
- Added backend code to honor that flag

## How to test this change

- Start MINT locally
- Sign in with your EUA & Okta
- Create a new model plan
- Ask a question on the model plan
- Note that it should be from an assessment team user
- Edit the [flag targeting settings in the local environment](https://app.launchdarkly.com/mint/local/features/downgradeAssessmentTeam) to target your own user.
  - Pro tip, use this snippet to get your LaunchDarkly user hash from your EUA!
```bash
$ echo -n "BC0V" | sha256sum | awk '{print $1}'
d5b58d45ed52bd0f91e753b607353111e4cf165c79822988b41bc04ee5a08889
```
- Ask another question on the model plan
- Note that it should _NOT_ be from an Assessment team user!

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
